mri/tests/test_proximity.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+# pySAP - Copyright (C) CEA, 2017 - 2018
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+##########################################################################
+
+# System import
+import unittest
+import numpy as np
+
+# Package import
+from mri.operators import WaveletN
+from mri.operators.proximity.weighted import WeightedSparseThreshold
+
+
+class TestProximity(unittest.TestCase):
+    # Test proximity operators
```

Comment:
I think it could be nice to add at least one thing, verify that the application of `op` runs smoothly.

The cherry on top would be to verify that the results make sense in a very small scale example.

Reply:
Haha, will see what i can do.. Although, in my opinion that is checked in modopt...

Reply:
Added very minute test. As I stated above the other tests will be done in Modopt as this is just a wrapper on Sparse Threshold.